tests/network/vmi_multus.go

```diff
@@ -1485,3 +1535,32 @@ func cloudInitNetworkDataWithStaticIPsByDevice(deviceName, ipAddress string) str
 	ExpectWithOffset(1, err).ToNot(HaveOccurred(), "should successfully create static IPs by device name cloud init network data")
 	return networkData
 }
+
+func configureAlpineInterface(vmi *v1.VirtualMachineInstance, ifaceName, staticIP string) error {
+	if staticIP == "" {
+		return configureAlpineInterfacesWithDynamicAddress(vmi, ifaceName)
+	}
+
+	return configInterface(vmi, ifaceName, staticIP)
+}
+
+func configureAlpineInterfacesWithDynamicAddress(vmi *v1.VirtualMachineInstance, ifacesNames ...string) error {
```

Comment:
`activate/enableDHCPOnVMInterfaces` maybe ? Your proposal is quite ok as is, not sure this is worth it.